packages/oas-utils/src/helpers/httpMethods.ts

```diff
@@ -80,3 +80,7 @@ export const getHttpMethodInfo = (methodName: string) => {
     }
   )
 }
+
+/** Type guard which takes in a string and returns true if it is in fact an HTTPMethod */
+export const isHTTPMethod = (method: string): method is RequestMethod =>
```

Comment:
code style police: pls use `isHttpMethod` (not `isHTTPMethod`)

Reply:
great call